yarn.lock

```diff
@@ -13128,6 +13128,78 @@ __metadata:
   languageName: node
   linkType: hard
 
+"foxbox@workspace:.":
```

Comment:
There are still references to foxbox in the diff. Please make sure that they will not be applied on merging

Reply:
Weird bug! I fixed it now again!